src/packages/src/xroad/common/proxy/usr/share/xroad/db/serverconf-changelog.xml

```diff
@@ -15,5 +15,6 @@
   <include file="serverconf/7-servicedescription.xml" />
   <include file="serverconf/8-restauthorization.xml" />
   <include file="serverconf/9-rest-auth-refactoring.xml" />
+  <include file="serverconf/10-apikeys.xml" />
   <include file="serverconf/10-rest-service-type-rename.xml" />
```

Comment:
Two migrations now start with `10-`. Probably doesn't break anything but style issue.

Reply:
There is also some style issues in the documents, like versions 2.31 and 2.31-UI in the version table. I think we could fix the style issues after merge, to get back to working version control for rest ui development as soon as possible.

Reply:
will add separate jira ticket for this https://jira.niis.org/browse/XRDDEV-723?focusedCommentId=12716&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-12716